app/routes/users/components/Penalties.js

```diff
@@ -22,11 +25,17 @@ function Penalties({ penalties }: Props) {
       {penalties.map(penalty => {
         const word = penalty.weight > 1 ? 'prikker' : 'prikk';
         return (
-          <li key={penalty.id}>
+          <li key={penalty.id} style={{ paddingBottom: '10px' }}>
             <strong>
-              {penalty.weight} {word}:{' '}
+              Du har fått {penalty.weight} {word}
             </strong>
-            {penalty.reason}
+            <br />
```

Comment:
You can use `<p>`-tags instead of these breaks :smile:

Reply:
Might be a better solution, yes.

Reply:
Ahh, i see.. Does wrapping each section in a `div` work? Otherwise, breaktags are ok

Reply:
Wrapping in  `div` 's did indeed work.